prow/gangway/gangway_grpc.pb.go

```diff
@@ -0,0 +1,187 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
I think we can make git/GH ignore this with gitattributes.

Reply:
Fixed locally.